alts/shared/models.py

```diff
@@ -26,11 +26,18 @@ class AsyncSSHParams(BaseModel):
     disable_known_hosts_check: bool = False
 
 
+class TestConfiguration(BaseModel):
+    tests: typing.Optional[typing.List[dict]]
+    test_env: typing.Optional[dict]
+
+
 class TaskRequestPayload(BaseModel):
     runner_type: str = 'any'
     dist_name: str
     dist_version: typing.Union[str, int]
     dist_arch: str
+    package_channel: typing.Optional[str] = None
```

Comment:
you can get rid of validation for `package_channel` field if you do the following:
```python
package_channel: typing.Optional[typing.Literal[*constants.ALLOWED_CHANNELS]]
```

Reply:
> Unpacked arguments cannot be used in type argument lists
> Unpack operator in subscript requires Python 3.11 or newer

`Pyright`

Reply:
you're right, but anyway, you can get rid of validation using `typing.Literal['stable', 'beta']`